frontend/pages/model/[modelId]/release/[semver]/review.tsx

```diff
@@ -0,0 +1,160 @@
+import { ArrowBack } from '@mui/icons-material'
+import { Box, Button, Container, Divider, Grid, Paper, Stack, Tooltip, Typography } from '@mui/material'
+import { useGetModel } from 'actions/model'
+import { useGetRelease, useGetReleasesForModelId } from 'actions/release'
+import { postReviewResponse, useGetReviewRequestsForModel } from 'actions/review'
+import { useGetUiConfig } from 'actions/uiConfig'
+import Markdown from 'markdown-to-jsx'
+import { useRouter } from 'next/router'
+import prettyBytes from 'pretty-bytes'
+import { useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReviewWithComment, { ResponseTypeKeys } from 'src/common/ReviewWithComment'
+import UserDisplay from 'src/common/UserDisplay'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+import Link from 'src/Link'
+import MessageAlert from 'src/MessageAlert'
+import CodeLine from 'src/model/registry/CodeLine'
+import Wrapper from 'src/Wrapper'
+import { mutate } from 'swr'
+import { formatDateString } from 'utils/dateUtils'
+import { getErrorMessage } from 'utils/fetcher'
+
+export default function ReleaseReview() {
+  const router = useRouter()
+  const { modelId, semver }: { modelId?: string; semver?: string } = router.query
+
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const { model, isModelLoading, isModelError } = useGetModel(modelId)
+  const { release, isReleaseLoading, isReleaseError } = useGetRelease(modelId, semver)
+  const { uiConfig, isUiConfigLoading, isUiConfigError } = useGetUiConfig()
+  const { mutateReleases } = useGetReleasesForModelId(modelId)
+  const { mutateReviews } = useGetReviewRequestsForModel({
+    modelId,
+    semver: semver as string,
```

Comment:
Shouldn't be asserting like this. The type of `string | undefined` is correct. We should instead ensure it is a string:
```
semver: `${semver}`
```